### DISCUSSION TITLE
How to classify the class names?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [tailwindlabs/tailwindcss#9859] [Help]
How to classify the class names?

Such as there is a div with two type classes:
```
<p>
  <a href="https://tailwindcss.com/docs" class="text-sky-500 hover:text-sky-600" >Read the docs &rarr;</a>
</p>
```
I want to classify the classes:
```
<p>
  <a href="https://tailwindcss.com/docs" class="text-sky-500 " class="hover:text-sky-600">Read the docs &rarr;</a>
</p>
```

but this do not work. 

I could use below way to do it:
```
<p>
  <a href="https://tailwindcss.com/docs" class="text-sky-500 ; hover:text-sky-600" >Read the docs &rarr;</a>
</p>
```
but it must use spaces to separate the class names. 

so, is there any succinct way to achieve my goal？ 
---

**Edit-01**

I think the below classes are different types classes:
```
class="text-sky-500 " 
class="hover:text-sky-600"
```
so I want to distinguish them out.

**Comment by wongjn** [comment]
Could you give more details about what you mean by "classify" please?
